osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -282,14 +282,15 @@ protected override bool OnClick(ClickEvent e)
             return true;
         }
 
-        private class PlaylistDownloadButton : BeatmapPanelDownloadButton
+        private sealed class PlaylistDownloadButton : BeatmapPanelDownloadButton
         {
             private readonly PlaylistItem playlistItem;
 
             [Resolved]
             private BeatmapManager beatmapManager { get; set; }
 
-            public override bool IsPresent => base.IsPresent || Scheduler.HasPendingTasks;
+            // required for download tracking, as this button hides itself. can probably be removed with a bit of consideration.
+            public override bool IsPresent => true;
```

Comment:
https://drive.google.com/file/d/1m7aAbV0Q0xUpKud9mHAuMsiSehbJwb4w/view?usp=sharing

The button is clickable even when it's hidden, making this feel pretty bad... I think this needs to be fixed before merge.

Reply:
Any suggestions on how to handle this better? I'd like to avoid having that there in the first place if possible.. Maybe moving the download tracker up one level?

Reply:
Actually the fading can probably be moved to the `shakeContainer`, will try that first.

Reply:
Ended up fixing this by adding a transition on the width change. Looks pretty cool while also fixing the issue with minimal changes.